go/vt/vtgate/buffer/flags.go

```diff
@@ -23,60 +23,50 @@ import (
 	"strings"
 	"time"
 
+	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/topo/topoproto"
 )
 
 var (
-	enabled       = flag.Bool("enable_buffer", false, "Enable buffering (stalling) of primary traffic during failovers.")
-	enabledDryRun = flag.Bool("enable_buffer_dry_run", false, "Detect and log failover events, but do not actually buffer requests.")
+	f_enabled       = flag.Bool("enable_buffer", false, "Enable buffering (stalling) of primary traffic during failovers.")
```

Comment:
This seems to introduce a new naming convention for flags. I'd prefer we don't do that in this PR. If we agree on such a convention we should change all flags in one go.
Feel free to rename them to be more descriptive, e.g. window -> bufferWindow.